attributes/network/cddl/9_account.cddl

```diff
@@ -131,16 +131,16 @@ account.info@param = {
     0 => account-identity,
 }
 account.info@return = {
-    ; The name or description of the wallet.
+    ; The name or description of the account.
     ? 0 => tstr,
 
     ; The list of roles for the account.
-    ? 1 => {
-        * non-anonymous-identity => [ * account-role ],
+    1 => {
+        1* non-anonymous-identity => [ * account-role ],
     },
 
     ; The list of features and their argument.
-    ? 2 => [ 1* account-feature ],
+    2 => [ 1* account-feature ],
```

Comment:
See my comment in `many-rs` : https://github.com/the-lifted-initiative/many-rs/pull/51#discussion_r856445297

WDYT?